viite-UI/src/model/ProjectCollection.js

```diff
@@ -260,7 +260,8 @@
                 if (successObject.errorMessage) {
                   new ModalConfirm(successObject.errorMessage);
                 }
-                validateUnchangedInProject(dataJson.projectId);
+                if (!successObject.projectErrors)
```

Comment:
// TODO This logic belongs to backend
comment here maybe

Reply:
Tämä sovittiin refaktoroitavan siten että Ennallaan toimenpiteen linkit validoidaan etäisyyslukemien päivittämisen yhteydessä, mutta kuitenkin ennen itse "laskentaa"
(jotta laskentaan ei päädytä tilanteessa jossa Ennallaan olevat linkit ovat tilassa joka rikkoisi laskennan)

Reply:
Eli logiikka siirtyy pois tästä tallenna/päivitä projektilinkit toiminnasta -> etäisyyslukemien päivityksen / muiden validaatioiden kanssa samaan kimppuun